web/impact/impact/v1/helpers/judge_role_criterion_helper.py

```diff
@@ -8,10 +8,12 @@
 class JudgeRoleCriterionHelper(JudgeCriterionHelper):
     cache_judge_field = "expertprofile__expert_category__name"
     judge_field = "judge__" + cache_judge_field
+    cache_key = cache_judge_field
 
     def analysis_annotate_fields(self):
         return {
             "role": F(self.judge_field),
+            self.cache_key: F(self.judge_field),
```

Comment:
also, the logic were adding here for example is querying the exact same data but with a different name

Reply:
I'm not sure yet what purpose "role" serves here. I'd love to be able to remove it, but this is a piece of the complexity that I have not yet been able to reduce

Reply:
ah, okay. so in all of these `analysis_annotate_fields` functions. The keys are used in the `analysis_tally` functions to retrieve the value from this [cache](https://github.com/masschallenge/impact-api/blob/0063180e4f8b6a85bdd82d7d5a1f8434907df0f7/web/impact/impact/v1/classes/option_analysis.py#L185)